x/lockup/keeper/lock.go

```diff
@@ -48,18 +48,21 @@ func (k Keeper) BeginUnlockAllNotUnlockings(ctx sdk.Context, account sdk.AccAddr
 }
 
 // AddToExistingLock adds the given coin to the existing lock with the same owner and duration.
-// Returns an empty array of period lock when a lock with the given condition does not exist.
-func (k Keeper) AddToExistingLock(ctx sdk.Context, owner sdk.AccAddress, coin sdk.Coin, duration time.Duration) ([]types.PeriodLock, error) {
+// Returns an empty struct of period lock when a lock with the given condition does not exist.
```

Comment:
Also, what is returned when a lock does exist? Consider elaborating please

Reply:
done!